### PR TITLE
fix(lidar_apollo_instance_segmentation): fix critical bug

### PR DESCRIPTION
**Pull request by kminoda**
## Description

Original code included a critical bug which caused an empty output from apollo node.

## Related links

**Parent Issue:**

- Link

## How was this PR tested?

We've tested with
- locally running the perception inference with apollo, and confirmed that the node outputs some estimation

Note that this doesn't fix the issues we had internally: https://evaluation.tier4.jp/evaluation/reports/2921e69d-65b2-5067-af4b-8fffa4166d74?project_id=prd_jt

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Comment by knzo25** [comment]
:bow:

File: launch/tier4_perception_launch/launch/perception.launch.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
   <arg name="input/pointcloud" default="/sensing/lidar/concatenated/pointcloud" description="The topic will be used in the detection module"/>
   <arg name="mode" default="camera_lidar_fusion" description="options: `camera_lidar_radar_fusion`, `camera_lidar_fusion`, `lidar_radar_fusion`, `lidar` or `radar`"/>
   <arg name="data_path" default="$(env HOME)/autoware_data" description="packages data and artifacts directory path"/>
-  <arg name="lidar_detection_model" default="centerpoint" description="options: `transfusion`, `centerpoint`, `apollo`, `pointpainting`, `clustering`"/>
+  <arg name="lidar_detection_model" default="apollo" description="options: `transfusion`, `centerpoint`, `apollo`, `pointpainting`, `clustering`"/>
```
[the remark this answers is not in the document]
Comment (kminoda) [reply launch/tier4_perception_launch/launch/perception.launch.xml]: Thanks, that's a mistake. Fixed with the latest version.

**Review by technolojin — APPROVED** [review]
LGTM

**Comment by kminoda** [comment]
CI keeps failing :cry:

**Review by Shin-kyoto — APPROVED** [review]
LGTM

- I confirmed that this PR content is good to merge.
   - I checked the CIs passed except `build-and-test-differential / clang-tidy-differential (pull_request)`
- I did **NOT** check this PR by running autoware.

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**